Makefile

```diff
@@ -17,6 +17,7 @@ npm-test :
 
 .PHONY : docs
 docs :
+	mkdir docs
```

Comment:
IIRC this will fail if `docs/` already exists, you need `mkdir -p docs` to avoid that.

Reply:
I need to stop working late at night. Thanks for catching this 🙌